src/main/app/src/app/zoeken/model/zoek-parameters.ts

```diff
@@ -6,6 +6,7 @@
 import {SorteerVeld} from './sorteer-veld';
 
 export class ZoekParameters {
+    type: 'ZAAK' | 'TAAK' | null;
```

Comment:
Ik weet het niet zeker maar moet hier niet ook DOCUMENT bij? omdat ik zag dat er in zoek.component.html al een verwijzing staat naar DOCUMENT

Reply:
documenten is minimaal geimplementeerd in angular, zal de resterende code ook opruimen